Error in typings: Cannot find name 'Stream'.

```
node_modules/@most/core/type-definitions/combinator/variadic.d.ts:6:19 - error TS2304: Cannot find name 'Stream'.

6   [K in keyof A]: Stream<A[K]>
                    ~~~~~~


Found 1 error.
```

--- a/packages/core/type-definitions/combinator/variadic.d.ts
+++ b/packages/core/type-definitions/combinator/variadic.d.ts
@@ -1,3 +1,5 @@
+import { Stream } from '@most/types'
+
 // Map arrays to arrays of Streams:
 // Array<A> => Array<Stream<A>>
 // [A, B, C, ...] => [Stream<A>, Stream<B>, Stream<C>, ...]

